src/service-bus-service.ts

```diff
@@ -0,0 +1,36 @@
+import { Guid } from 'guid-typescript';
+import { AzureWebCommandService } from './azure-web-command-service';
+
+export class ServiceBusService {
+    private baseUrl: string;
+    private sharedAccessKey: string;
+
+    constructor(sbNameSpace: string,
+                sharedAccesKey: string) {
+        this.baseUrl = sbNameSpace;
+        this.sharedAccessKey = sharedAccesKey;
+    }
+
+    public async sendMessageAsync(queueName: string, correlationId: Guid, args?: any[]): Promise<void> {
+        const url = `${this.baseUrl}/ReceiveWebEvent`;
+        try {
+            // First we add it to ensure that the id is stored before the response is received.
+            AzureWebCommandService.addCommand(queueName, correlationId);
```

Comment:
Looking at it now it's not clear that the AzureWebCommandService is for local storage. Renaming the service so it describes that would be better. AzureWebCommandService could be very be the name of sending the Azure Web Commands to the backend as well. You won't need the comment then.

Reply:
I think this line also belongs outside the try-scope as you're not expecting an issue here.

Reply:
Changed the class to LocalCommandStorageService, this gives a better understanding of the class responsibility